doc/gitian-building.md

```diff
@@ -15,12 +12,19 @@ These results are compared and only if they match, the build is accepted and pro
 for download.
 
 More independent Gitian builders are needed, which is why this guide exists.
-It is preferred you follow these steps yourself instead of using someone else's
-VM image to avoid 'contaminating' the build.
+
+# Gitian building using Docker (Current)
+Veil gitian builds utilize docker images and containers. More information about that process can be found here:
+https://github.com/Veil-Project/docker-based-gitian-builder
+
+
+# Gitian building using a VM (Legacy)
+*Maintained for legacy purposes. The docker process above should be used for all new builds.*
+- *Setup instructions for a Gitian build of Veil using a VM or physical system.*
+- *You Should use Ubuntu 18.04.1 server for the VM host until further notice.*
 
 Table of Contents
 ------------------
-
 - [Preparing the Gitian builder host](#preparing-the-gitian-builder-host)
```

Comment:
Check these links.  Looks like the first one jumps to the section but the other sections don't seem to point to the right places.

Reply:
Updated links. Removed duplicate file.